lib/MetaCPAN/Script/CVE.pm

```diff
@@ -179,7 +179,10 @@ sub index_cve_data {
 
                 if ( $releases->{hits}{total} ) {
                     ## no critic (ControlStructures::ProhibitMutatingListFunctions)
-                    @matches = sort { $a->{version} <=> $b->{version} }
+                    @matches = sort {
+                        numify_version( $a->{version} )
+                            <=> numify_version( $b->{version} )
+                        }
```

Comment:
maybe consider a local cache for `numify_version` or not 🤷

Reply:
sure, but I didn't think the runtime saved would be worth the extra memory used. this script takes <1m. :)

Reply:
@atoomic please check the newer version